WORKSPACE

```diff
@@ -10,59 +10,94 @@ load("@io_bazel_rules_go//go:def.bzl", "go_repositories")
 
 go_repositories()
 
+# To build the par file for the deb_file_loader helper program in deb_helper
+git_repository(
+    name = "subpar",
+    remote = "https://github.com/google/subpar",
+    tag = "1.0.0",
+)
+
+# An example of actually using the new rules of this project:
+
+# Load the rules
 load(
-    "//package_manager:package_manager.bzl",
-    "package_manager_repositories",
-    "dpkg_src",
-    "dpkg_list",
+    "//deb_helper:deb_helper.bzl",
+    "deb_loader",
+    "deb_packages",
 )
 
-package_manager_repositories()
+# Initialize the deb_loader which downloads the deb files in deb_packages rules
+deb_loader()
 
-dpkg_src(
-    name = "debian_jessie",
-    arch = "amd64",
+# For every source of deb packages create a deb_packages rule
+# The name does not have to be like the one here, but it helps to know which source is used in BUILD files
+deb_packages(
+    name = "debian_jessie_amd64",
+    architecture = "amd64",
     distro = "jessie",
-    sha256 = "142cceae78a1343e66a0d27f1b142c406243d7940f626972c2c39ef71499ce61",
-    snapshot = "20170821T035341Z",
-    url = "http://snapshot.debian.org/archive",
+    distro_type = "debian",
+    mirrors = [
+        "http://deb.debian.org",
```

Comment:
Typically mirrors are specified up to the root folder in `sources.list` files (e.g. `deb http://deb.debian.org/debian unstable main`). For now I went with the simpler way this was done in this project before, but it might be the case that this has to change in the future.